src/transformers/trainer.py

```diff
@@ -656,6 +666,7 @@ def create_optimizer_and_scheduler(self, num_training_steps: int):
             else:
                 self.optimizer = optimizer_cls(optimizer_grouped_parameters, **optimizer_kwargs)
 
+    def create_scheduler(self, num_training_steps: int):
```

Comment:
If we are making these public methods, this one  would need a docstring too.

Reply:
Added docstring